apps/workspaces/migrations/0059_remove_sageintacctcredential_access_token_and_more.py

```
@@ -0,0 +1,25 @@
+# Generated by Django 4.2.27 on 2025-12-11 03:43
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('workspaces', '0058_intacctcompanytoken_and_more'),
+    ]
```

Ruff (0.14.8)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)